hatch/src/project/compiler_options.rs

```diff
@@ -0,0 +1,67 @@
+use platform::arch::Arch;
+use project::Target;
+use project::ProjectKind;
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct CompilerOptions {
+  compiler: String,
+  compiler_flags: String,
+  linker_flags: String,
+  arch: Arch,
+  target: Target,
+}
```

Comment:
A serializable struct for compiler options, intended to be put in the hatch file